### ISSUE TITLE
VRChat crashes in VR mode since new 3.16.1 beta

### ISSUE THREAD

**Issue by elypter** [ValveSoftware/Proton#1740]
# Compatibility Report
- Name of the game with compatibility issues: VRChat
- Steam AppID of the game: 438100

## System Information
- GPU: RX 580
- Driver/LLVM version: Mesa 18.2.2
- Kernel version: 4.18.12-steamvr-generic #steamvr1-Ubuntu SMP Fri Oct 5 00:22:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- Link to full system information report as [Gist](https://gist.github.com/): 
https://gist.github.com/elypter/6178645b417dea592d971378a2cecb2d
- Proton version: 3.16.1 beta

## I confirm:
- [X ] that I haven't found an existing compatibility report for this game.
- [X ] that I have checked whether there are updates for my system available.

couldnt find a steam-$APPID.log

## Symptoms

when i launch vrchat with the new proton 3.16.1 beta in vr mode i get this error message. it doesnt happen in desktop mode and it doesnt happen in vr mode with proton 3.7-8

![bildschirmfoto von 2018-10-15 03-40-36](https://user-images.githubusercontent.com/2572458/46925516-84cd8500-d02c-11e8-8ef7-ce3345f178b8.png)

[error.log](https://github.com/ValveSoftware/Proton/files/2477132/error.log)
[2018-10-15_034012.zip](https://github.com/ValveSoftware/Proton/files/2477139/2018-10-15_034012.zip)

## Reproduction

i use the launch options "PROTON_LOG=1 vrchat://launch?id=wrld_69ab9cdf-5436-46bd-98b5-714837a53b4f -force-d3d11 -show-screen-selector"

i start steamvr before starting vrchat

**Comment by kisak-valve** [comment]
Hello @elypter, this issue is already being tracked at #1731. Closing in favor of the existing issue report.
